recipes/pycudadecon/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pycudadecon" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+
+  url: https://github.com/tlambert03/pycudadecon/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f0e15408724b8eb88581fba8bc8b5dc93f107d2c4e061b6007c9bd773ab93dea
+
+build:
+  skip: True  # [osx]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
```

Comment:
```suggestion
    - python >=3.7
```

Reply:
oh sorry... just saw the lint error [above](https://github.com/conda-forge-linter) ... maybe ignore this one... (but I don't intend to support <3.7)